docs/reference/query-dsl/sparse-vector-query.asciidoc

```diff
@@ -0,0 +1,278 @@
+[[query-dsl-sparse-vector-query]]
+== Sparse vector query
+
+++++
+<titleabbrev>Sparse vector</titleabbrev>
+++++
+
+The sparse vector query executes a query consisting of sparse vectors, such as built by a learned sparse retrieval model.
+This can be achieved with one of two strategies:
+
+- Using an {nlp} model to convert query text into a list of token-weight pairs
+- Sending in precalculated token-weight pairs as query vectors
+
+These token-weight pairs are then used in a query against a <<sparse-vector,sparse vector>>.
```

Comment:
Could you add a section on scoring? 

That the scoring is effectively a dot-product between the stored dimensions and the query dimensions.

Reply:
Added, let me know what you think!